Revision en docs (#1323)
mercadopago/devsite-docs: guides/plugins/unofficial/wix.en.md

>
>Important
>

> >Do you want to learn more about credentials? Find more information under [credentials](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/resources/credentials).

Reviewer: ```suggestion
> For more information about credentials, access the [credentials](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/resources/credentials) documentation.
```
